src/test/transaction_tests.cpp

```diff
@@ -887,8 +898,8 @@ BOOST_AUTO_TEST_CASE(txsize_activation_test) {
     CTransaction minTx;
     TxValidationState state;
 
-    BOOST_CHECK(!ContextualCheckTransaction(
-        params, minTx, state, 0, 5678, 1234));
+    BOOST_CHECK(
```

Comment:
Would it be possibly to patch add the changes without these formatting adjustments?

Reply:
Will do.